test/test_ssltransport.py

```diff
@@ -229,8 +230,16 @@ def test_ssl_object_attributes(self) -> None:
             assert ssock.selected_npn_protocol() is None
 
             shared_ciphers = ssock.shared_ciphers()
-            assert type(shared_ciphers) == list
-            assert len(shared_ciphers) > 0
+            # https://github.com/python/cpython/issues/96931
```

Comment:
Since we don't *really* think this test will ever fail, just to cover the lines where we're forwarding attributes, can we simplify this test case to not have all the versions? ie `assert shared_ciphers is None or (type(shared_ciphers) is list and len(shared_ciphers) > 0)`?

Reply:
Hmm, I was thinking we'd want to implement whatever the behavior is for that Python version, even if that behavior isn't correct by the docs?

Reply:
Yeah there's a lot of options here, can I hand this PR over to you?